sound/soc/sof/ipc4.c

```diff
@@ -0,0 +1,669 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2022 Intel Corporation. All rights reserved.
+//
+// Authors: Rander Wang <rander.wang@linux.intel.com>
+//	    Peter Ujfalusi <peter.ujfalusi@linux.intel.com>
+//
+#include <sound/sof/header.h>
+#include <sound/sof/ipc4/header.h>
+#include "sof-priv.h"
+#include "sof-audio.h"
+#include "ops.h"
+
+#ifdef DEBUG_VERBOSE
+#define sof_ipc4_dump_payload(sdev, ipc_data, size)		\
+		print_hex_dump(KERN_DEBUG, "Message payload: ",	\
+			       DUMP_PREFIX_OFFSET,		\
+			       16, 4, ipc_data, size, false)
+#else
+#define sof_ipc4_dump_payload(sdev, ipc_data, size)	do { } while (0)
+#endif
```

Comment:
why not just use print_hex_dump_debug without this DEBUG_VERBOSE?

Reply:
True, I should have  used that version.